src/index.html

```diff
@@ -1,12 +1,298 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo-moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#">Smartphones</a>
+          </li>
```

Comment:
don`t forget to add empty line between sibling tags
![2023-02-14_13h06_09](https://user-images.githubusercontent.com/61904995/218718961-8c03253d-497f-417a-bac7-d14d61dc57b6.png)